webpack.mix.js

```diff
@@ -24,25 +24,22 @@ mix.setPublicPath('./dist');
 mix.browserSync({
   proxy: 'https://example.test',
   files: [
-    'app/**/*.php',
-    'config/**/*.php',
-    'resources/views/**/*.php',
-    'dist/styles/**/*.css',
-    'dist/scripts/**/*.js'
+    '(app|config|resources)/**/*.php',
+    publicPath`(styles|scripts)/**/*.(css|js)`,
   ],
 });
 
 // Styles
-mix.sass(src('styles/app.scss'), 'styles');
+mix.sass(src`styles/app.scss`, 'styles');
```

Comment:
Still not feeling the idea of dropping the parenthesis. It makes it a lot less natural / readable in my opinion. Would like to hear others thoughts!